src/raster/r.estimap.recreation/r.estimap.recreation/r.estimap.recreation.md

```diff
@@ -701,7 +701,7 @@ This time it works. Here the first few lines from the output CSV files:
 
 returns
 
-```sh
+```csv
```

Comment:
Wanna improve the content here to make it actual csv?

Reply:
it's output from head command, so i marked it as text